sharedTestCode/src/main/java/org/simple/sharedTestCode/TestData.kt

```diff
@@ -1519,13 +1519,13 @@ object TestData {
                       id = "blood_pressure_checks_male",
                       linkId = "monthly_screening_reports.blood_pressure_checks_male",
                       text = "Male",
-                      type = Integer,
+                      type = IntegerType,
                       validations = InputFieldValidations(min = 0, max = 1000000)),
                   InputFieldComponentData(
                       id = "blood_pressure_checks_female",
                       linkId = "monthly_screening_reports.blood_pressure_checks_female",
                       text = "Female",
-                      type = Integer,
+                      type = IntegerType,
                       validations = InputFieldValidations(min = 0, max = 1000000))
```

Comment:
Can we add test data for Paragraph, Unordered List, String Input and Radio Button View group?